client/blocks/reader-post-card/style.scss

```diff
@@ -178,6 +179,30 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 	flex-wrap: wrap;
 	flex-direction: row;
 
+	.reader-post-card__featured-image {
+		flex-basis: auto;
+		flex-grow: 1;
+		margin-right: 20px;
+		max-width: 190px;
+
+		@include breakpoint( ">960px" ) {
+			max-width: 250px;
+		}
+
+		@media #{$reader-post-card-breakpoint-medium} {
+
+			ight: 100px;
```

Comment:
Good catch @samouri! Removed that block in https://github.com/Automattic/wp-calypso/pull/8865/commits/e3f73e3115603b7a03ed25156c3d4489557fa9a1. They apparently weren't needed.